src/journal/managed-journal-file.c

```diff
@@ -106,9 +119,10 @@ static int managed_journal_file_punch_holes(JournalFile *f) {
         sz = le64toh(f->header->data_hash_table_size);
 
         for (uint64_t i = p; i < p + sz && n > 0; i += n) {
-                n = pread(f->fd, items, MIN(sizeof(items), p + sz - i), i);
+                size_t m = MIN(sizeof(items), p + sz - i);
```

Comment:
Nit but the variable is a bit superfluous now that we only use it in one place anymore.